GitExtUtils/FileLogging/LogFilenameCreator.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.IO;
+
+namespace GitExtUtils.FileLogging
+{
+    public class LogFilenameCreator
+    {
+        private readonly string _logName;
+        public readonly string LogPath;
+
+        private Tuple<DateTime, string> _result;
+
+        public LogFilenameCreator(string logPath, string logName)
+        {
+            LogPath = logPath;
+            _logName = logName;
+            _result = Renew();
+        }
+
+        private Tuple<DateTime, string> Renew()
```

Comment:
[style] in general we strive to order the code as per [these guidelines](https://blogs.msdn.microsoft.com/brada/2005/01/26/internal-coding-guidelines/#_Toc6325250)

Reply:
What am I breaking? Its not clear from the link.